apps/src/codebridge/Workspace/HeaderButtons.tsx

```diff
@@ -45,6 +66,19 @@ const WorkspaceHeaderButtons: React.FunctionComponent = () => {
           />
         </WithTooltip>
       )}
+      {appName === 'pythonlab' && skipUrl && (
```

Comment:
For now, this 'Skip to project' button is an option in pythonlab only.

Reply:
I think it's fine to have this enabled more generally, right now the only other lab using this component is Web Lab 2, which isn't used. In theory we could want the same thing in web lab 2 eventually, and since no web lab 2 levels have a `skipUrl` there's no risk to removing `appName === 'pythonlab'` here.